test/jdk/javax/swing/SwingUtilities/TestBadBreak/TestBadBreak.java

```diff
@@ -40,7 +40,7 @@
  * @bug 8015085 8079253
  * @summary Shortening via " ... " is broken for Strings containing a combining
  *          diaeresis.
- * @run main TestBadBreak
+ * @run main/othervm -Dsun.java2d.uiScale=1 TestBadBreak
```

Comment:
I missed the point why the hardcoded values do not work in this test, both frame and robot should take the coordinates in the user's space.

Reply:
It uses hardcoded 200,90 for frame.setBounds.I guess it should be multiplied by scale factor for correct bounds to be used.

Reply:
Both coordinates robot/frame use the user's space and internally should be transformed to the device space. But that transformation should be the same, there are should not be differences in the final result for both.

Reply:
BTW the test itself draws the button to the BufferedImage so uiscale should not affect that rendering, both subtests should work similarly.

Reply:
I believe similar thing was done for https://github.com/openjdk/jdk/pull/950/files where the panel was drawn to bufferedimage but uiScale was set to 1 to make the test stable and robot was also used there to just delay but setUndecorated and setLocationRelativeTo() was used.

Reply:
That test uses a robot, and it may fail if the windows appeared under dock/etc.